webhooks-extension/src/components/WebhookDisplayTable/WebhookDisplayTable.js

```diff
@@ -280,6 +294,23 @@ export class WebhookDisplayTable extends Component {
               />
             </div>
 
+            {initialRows.length === 0 && namespace === ALL_NAMESPACES && (
```

Comment:
Refactor to avoid duplicating the containers, it's only the strings that differ in these 2 cases. This will help to ensure consistency after any future updates as there's only a single instance of the container that needs to be updated.

```js
{initialRows.length === 0 && (
  <p className=...>
    {namespace === ALL_NAMESPACES ? 'No webhooks created under any ...' : 'No webhooks created under namespace ...'}
  </p>
)}
```